### PR TITLE
ChannelFilter

### PR DESCRIPTION
**Pull request by dzucconi**
Trying to deal with typing forwardRef:

```
src/v2/components/UI/SearchInput/index.tsx:204:4 - error TS2322: Type '{ query?: string; onFocus?: () => any; onBlur?: () => any; onQueryChange?: (props: any) => any; onDebouncedQueryChange?: (props: any) => any; debounceWait?: number; forwardedRef: any; iconMap?: IconMap; ... 407 more ...; zIndex?: ResponsiveValue<...>; }' is not assignable to type 'IntrinsicClassAttributes<SearchInput>'.
  Types of property 'ref' are incompatible.
    Type 'LegacyRef<HTMLDivElement>' is not assignable to type 'LegacyRef<SearchInput>'.
      Type '(instance: HTMLDivElement) => void' is not assignable to type 'LegacyRef<SearchInput>'.
        Type '(instance: HTMLDivElement) => void' is not assignable to type '(instance: SearchInput) => void'.
          Types of parameters 'instance' and 'instance' are incompatible.
            Type 'SearchInput' is missing the following properties from type 'HTMLDivElement': align, addEventListener, removeEventListener, accessKey, and 236 more.

204   <SearchInput forwardedRef={ref} {...props} />
       ~~~~~~~~~~~
```

### CODE REVIEW

File: src/v2/components/UI/SearchInput/index.tsx [hunk 1]
```diff
@@ -193,6 +200,7 @@ class SearchInput extends PureComponent {
   }
 }
 
-export default React.forwardRef((props, ref) => (
-  <SearchInput forwardRef={ref} {...props} />
+export default React.forwardRef((props: Props, ref) => (
+  // @ts-ignore
```
Comment (dzucconi) [inline src/v2/components/UI/SearchInput/index.tsx]: I'm going to ignore this for the time being

File: src/v2/components/UI/Box/index.ts [hunk 2]
```diff
@@ -64,6 +85,30 @@ export const mixin = css`
...
+    BorderColorProps,
+    BorderRadiusProps,
+    BgColorProps,
+    TextAlignProps,
+    FlexProps,
+    TopProps,
+    RightProps,
+    BottomProps,
+    LeftProps,
+    ZIndexProps {}
```
Comment (damassi) [inline src/v2/components/UI/Box/index.ts]: 👍

Comment (damassi) [reply src/v2/components/UI/Box/index.ts]: Super minor, but this goes a long way for something that could potentially have many more additions: 

Screen Shot 2019-05-07 at 10 30 10 AM

Comment (dzucconi) [reply src/v2/components/UI/Box/index.ts]: yes plzz

**Pull request merged**